java/src/main/resources/arguments.yaml

```diff
@@ -28,10 +28,15 @@ ACCESS_TOKEN: null
 # =========================
 # Optional configurations:
 # =========================
-# Topic to publish/subscribe to (default: /event/CarMaintenance__e)
+# Topic to publish/subscribe to (default: /event/Order_Event__e)
 TOPIC: null
-# Number of Events to publish (default: 5)
-NUMBER_OF_EVENTS_TO_PUBLISH: null
+# Number of Events to publish in single or separate batches (default: 5)
+# Used only by Publish.java and PublishStream.java
+NUMBER_OF_EVENTS_TO_PUBLISH: 3
```

Comment:
The yaml file should have default values. Maybe use those values instead?

NUMBER_OF_EVENTS_TO_PUBLISH: 5
SINGLE_BATCH: false